src/olympia/blocklist/tests/test_admin.py

```diff
@@ -304,6 +313,14 @@ def test_add_single(self):
             f'The blocklist submission {FANCY_QUOTE_OPEN}No Sign-off: guid@; '
             f'dfd; some reason{FANCY_QUOTE_CLOSE} was added successfully.'
         ]
+        # The disabled and deleted versions should only have the block activity,
+        # not a reject activity
+        log_disabled_version = ActivityLog.objects.for_versions(disabled_version)
+        assert len(log_disabled_version) == 1
+        assert log == log_disabled_version.last()
+        log_deleted_version = ActivityLog.objects.for_versions(deleted_version)
+        assert len(log_deleted_version) == 1
+        assert log == log_deleted_version.last()
```

Comment:
could you change the tests here to be in a similar style as the previous comment?  (functionally they'll be the same, but it'd be more consistent)